src/openMVG/robust_estimation/rand_sampling.hpp

```diff
@@ -35,10 +35,11 @@
 namespace openMVG {
 namespace robust{
 
+// rperrot : is this legal ? why using an anonymous namespace here ? 
 namespace
 {
   std::default_random_engine random_generator;
```

Comment:
Curious that it is not supported since this is a c++11 feature http://en.cppreference.com/w/cpp/language/storage_duration
Perhaps will move the variable declaration to the function where it is used std::default_random_engine random_generator; Don't know the speed impact.